rewrite-yaml/src/test/java/org/openrewrite/yaml/CoalescePropertiesTest.java

```diff
@@ -287,6 +287,52 @@ void doNotCoalesceDocumentsHavingAnchorsAndAliases() {
         );
     }
 
+    @Test
+    void usingYamlTest() {
```

Comment:
I wonder if we really need these declarative tests at all. With these tests you are actually testing the Open Rewrite framework, not the options of the recipe. I think in this PR I would just add the `required = false` property and nothing else.